examples/misc/programs/misc/src/context.rs

```diff
@@ -54,15 +60,24 @@ pub struct TestPdaInit<'info> {
 #[derive(Accounts)]
 #[instruction(bump: u8)]
 pub struct TestPdaInitZeroCopy<'info> {
-    #[account(init, seeds = [b"my-seed".as_ref(), &[bump]], payer = my_payer)]
+    #[account(
+        init,
+        seeds = [b"my-seed".as_ref()],
```

Comment:
it's probably more readable to use that
```
pub const SEED: &str = "some-string-here";

   (...)
    seeds = [SEED.as_bytes()],
```
it's already supported by the framework, so why not use it in a couple of examples?